### DISCUSSION TITLE
ReVanced for non-android smart tv users?

### DISCUSSION THREAD

**Discussion by kaisen7887** [ReVanced/revanced-discussions#403] [General] [answered]
Any possibility to create a program (or vanced like program) for non android smart tv users (for ex:LG smart Tv)?

**Comment by reisxd** [comment]
I tried creating a YT TV client from scratch and well.. That would require too much time.

**Reply by kaisen7887** [reply]
> I tried creating a YT TV client from scratch and well.. That would require too much time.

Imean i can wait as long as possible (not for more than a year please) lol

**Comment by Sculas** [comment, marked as answer]
Support for YouTube TV is out of scope for ReVanced, this applies to Android TV _and_ non-android TV users.
